tests/Functional/Features/registration.feature

```diff
@@ -13,6 +13,30 @@ Feature: When an user needs to register for a new token
     And the SAML Response should contain element "StatusCode" with attribute "Value" with attribute value "urn:oasis:names:tc:SAML:2.0:status:Success"
     And the SAML Response should contain element "NameID" with value containing "test-user@institution-a.example.com"
 
+  Scenario: The user is presented the page in his preferred language based on stepup-locale cookie NL
```

Comment:
Is it possible to move this to a separate feature. For example to `locale.feature`  to make the location of the tests more clear.

Reply:
A good suggestion, addressed this in https://github.com/OpenConext/Stepup-Azure-MFA/pull/17/commits/c857b4a2d1e99aba8310c614ceb4d8145693192c